csharp/ql/test/query-tests/Security Features/CWE-079/XSS/Index.cshtml.g.cs

```diff
@@ -0,0 +1,62 @@
+// A hand-written test file that mimics the output of compiling a `.cshtml` file
+#pragma checksum "Index.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "c4ae76542f1958092cebd8f57beef899d20fc548"
+// <auto-generated/>
+#pragma warning disable 1591
+[assembly: global::Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemAttribute(typeof(dotnetweb.Pages.Pages_Index), @"mvc.1.0.razor-page", @"Index.cshtml")]
+namespace dotnetweb.Pages
+{
+    #line hidden
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+    using Microsoft.AspNetCore.Mvc.ViewFeatures;
+#nullable restore
+using dotnetweb;
+
+#line default
+#line hidden
+#nullable disable
+    [global::Microsoft.AspNetCore.Razor.Hosting.RazorSourceChecksumAttribute(@"SHA1", @"c4ae76542f1958092cebd8f57beef899d20fc548", @"Index.cshtml")]
+    // [global::Microsoft.AspNetCore.Razor.Hosting.RazorSourceChecksumAttribute(@"SHA1", @"c13da96c2597d5ddb7d415fb4892c644a268f50b", @"/Pages/_ViewImports.cshtml")]
+    public class Pages_Index : global::Microsoft.AspNetCore.Mvc.RazorPages.Page
+    {
+        #pragma warning disable 1998
+        public async override global::System.Threading.Tasks.Task ExecuteAsync()
+        {
+#nullable restore
+#line 3 "Index.cshtml"
+  
+    ViewData["Title"] = "ASP.NET Core";
+    var message = Request.Query["m"];
+
+#line default
+#line hidden
+#nullable disable
+            WriteLiteral("<div class=\"cli\">\n    <div class=\"cli-example\">        \n");
+#nullable restore
+#line 14 "Index.cshtml"
```

Comment:
It doesn't look like the Index.cshtml file contains any code that would have generated this (even though the "generated" code here i handwritten).

Reply:
The contents of the `.cshtml` file is not important; it is only added because the extractor would otherwise fail when it attempts to add the file to the source archive (because the file is referenced with a `#line` directive).

Reply:
Alright - understood. Maybe it is of less importance, but I was just worried that it might be confusing when revisited later that the content of the .cs file doesn't correspond to the content of the .cshtml file (even though it is stated that it is handwritten). One could consider, if we should add a comment in the .cshtml file with your statement from above.